bims/urls.py

```diff
@@ -85,4 +89,6 @@
         activate_user, name='activate-user'),
     url(r'^under-development/$',
         UnderDevelopmentView.as_view(), name='under-development'),
+    url(r'^nonvalidated-list/$',
```

Comment:
You should at least make this as a login_required page, so public can't access this

Reply:
done @dimasciput